app/models/planning_application.rb

```diff
@@ -423,6 +419,14 @@ def send_update_notification_to_reviewers
 
   private
 
+  def set_reference
```

Comment:
Possibly my obsession with not going over the guide line in Atom is out of control 😅 .